backend/editor/api.py

```diff
@@ -309,7 +376,13 @@ async def export_to_github(
 
 
 @app.post("/{taxonomy_name}/{branch}/import")
-async def import_from_github(request: Request, branch: str, taxonomy_name: str):
+async def import_from_github(
+    request: Request,
+    branch: str,
+    taxonomy_name: str,

```

Comment:
I think this part can be removed, and can be taken from the parameters variable.